### PR TITLE
Add content for Collector v0.38.0

### PR DESCRIPTION
**Pull request by alexperez52**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by alexperez52** [comment]
Let's wait until release is done

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+**Upstream changelog**
+
+* [OpenTelemetry Collector v0.93.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.93.0)
+* [OpenTelemetry Collector v0.94.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.94.0)
+* [OpenTelemetry Collector Contrib v0.93.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.93.0)
+* [OpenTelemetry Collector Contrib v0.94.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.94.0)
+
+**Release Notes**
+- Add release to fix CVE-2024-21626 [\#2643](https://github.com/aws-observability/aws-otel-collector/issues/2643)
+- Update go dependencies [\#2638](https://github.com/aws-observability/aws-otel-collector/pull/2638) ([bryan-aguilar](https://github.com/bryan-aguilar))
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx:26]:
```suggestion
- Update Otel and go dependencies [\#2638](https://github.com/aws-observability/aws-otel-collector/pull/2638) ([bryan-aguilar](https://github.com/bryan-aguilar))
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+**Notice**
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx:28]: I think we can remove this notice here or add `v0.38.0` as part of the notice

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+We are aware of a potential issue in the `runc` package (CVE-2024-21626), used by the AWS Distribution for Open Telemetry (ADOT) Collector up to and including v0.37.0.
+We have a new release of the ADOT Collector, v0.37.1 updating packages affected by this vulnerability. We recommend that users update their ADOT Collector
+to at least v0.37.1 at the earliest opportunity. See https://aws-otel.github.io/docs/getting-started/collector for information on deploying the ADOT Collector.
+
+**Download**
+
+Detailed technical documentation is available on the [ADOT developer site](https://aws-otel.github.io/),
+and you can [download the distribution](https://aws-otel.github.io/download) from
+[GitHub](https://github.com/aws-observability/aws-otel-collector/releases/tag/v0.37.0).
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx:38]:
```suggestion
[GitHub](https://github.com/aws-observability/aws-otel-collector/releases/tag/v0.38.0).
```

Comment (alexperez52) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx]: Fixed

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx (hunk 2 above)
Comment (alexperez52) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx]: Removed

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx (hunk 1 above)
Comment (alexperez52) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx]: Fixed

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+**Upstream changelog**
+
+* [OpenTelemetry Collector v0.93.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.93.0)
+* [OpenTelemetry Collector v0.94.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.94.0)
+* [OpenTelemetry Collector Contrib v0.93.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.93.0)
+* [OpenTelemetry Collector Contrib v0.94.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.94.0)
+
+**Release Notes**
+- Add release to fix CVE-2024-21626 [\#2643](https://github.com/aws-observability/aws-otel-collector/issues/2643)
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.38.0.mdx:25]: we can remove this, as it is mentioned in `v0.37.1`

**Comment by alexperez52** [comment]
Converting to draft until release is done

**Review by vasireddy99 — APPROVED** [review]

**Pull request merged**